### PR TITLE
Add Homebrew installation method to README

### PR DESCRIPTION
**Pull request by srvv**
It would be nice to add this option to documentation as it is easier in most cases. #161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,14 @@ directory.
 To install, just copy this one executable into a directory that is
 part of your `PATH` environment variable.
 
+### Alternatively install via Homebrew
+
+If you prefer using [Homebrew](https://brew.sh):
+
+```
+$ brew install swift-protobuf
+```
```
Comment (tbkka) [inline README.md]: I presume this only installs the command-line tool?

We should be more clear about that.

Comment (srvv) [reply README.md]: This is part of 'Building and Installing the Code Generator Plugin' section. Also described what this command does.

**Comment by srvv** [comment]
Fixed.

**Comment by tbkka** [comment]
Excellent!  Thanks!  😎

**Pull request merged**